### PR TITLE
Create ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3791](https://cumulocity.atlassian.net/browse/DM-3791)
Original PR: [6814](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6814)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Repository configuration saves now correctly configurationType. [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Repository configuration now correctly saves the configuration type
```

File: content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3791
+version: 1020.14.5
+---
+Repository configuration saves now correctly configurationType. [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the repository configuration, the configuration type was previously not saved correctly, leading to unexpected behavior. With this change, the configuration type is now properly persisted when saving the repository configuration. This ensures that the intended configuration type is used consistently, avoiding potential misconfiguration issues for users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3791
+version: 1020.14.5
+---
+In the repository configuration, the configuration type was not saved correctly when adding a new entry, leading to unexpected behavior. With this change, the configuration type is now properly persisted when saving the repository configuration.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md:17]: @dafkk Isn´t it "configuration repository" and "configuration snapshot"?

Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md:17]:
```suggestion
In the configuration repository, the configuration type was not saved correctly when adding a new configuration snapshot, leading to unexpected behavior. With this change, the configuration type is now properly persisted when saving the configuration snapshot. 
```

Comment (BeateRixen) [reply content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md]: Sorry, I accidentially commited my suggestion. Please check.

File: content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Repository configuration now correctly saves the configuration type
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md:3]:
```suggestion
title: Configuration repository now correctly saves the configuration type
```

File: content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md (hunk 3 above)
Comment (dafkk) [reply content/change-logs/device-management/ui-c8y-1020-14-5-repository-configuration-saves-now-correctly-configuration-type-graft-release-cd.md]: Yes yes I have checked and I think I committed it, all good thanks for the hint :)

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**